apis/k8ssandra/v1alpha1/k8ssandracluster_types.go

```diff
@@ -85,6 +85,9 @@ type K8ssandraClusterStatus struct {
 	//
 	// TODO Figure out how to inline this field
 	Datacenters map[string]K8ssandraStatus `json:"datacenters,omitempty"`
+
+	// +kubebuilder:default=None
+	Error string `json:"error,omitempty"`
```

Comment:
It would be nice to add some docs for this.

Reply:
Sure thing 👍